.github/workflows/tests.yml

```
@@ -0,0 +1,38 @@
+name: edx-platform tests
+
+on: [push]
```

This change make it just on PRs and uses the `merge-head` instead of the PR `HEAD`? What do you think?

```suggestion
on: [pull_request]
```

It means that we won't be able to build on `main` but I think we can configure that out later.